app/Console/Commands/CheckCommand.php

```diff
@@ -54,7 +61,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $paths = (array) $input->getArgument('paths');
 
         /** @var string[] $typesToSkip */
-        $typesToSkip = (array) $input->getOption('skip-type');
+        $typesToSkip = [];
+
+        if ($input->getOption('configuration') !== null) {
+            $config = Yaml::parseFile($input->getOption('configuration'));
+            $typesToSkip = $config['typesToSkip'] ?? [];
+        }
+
+        if (count($input->getOption('skip-type')) > 0) {
+            $typesToSkip = (array)$input->getOption('skip-type');
+        }
```

Comment:
Could you extract this section to new method? So its more clear to read.

Reply:
Done